content/blog/2021/announcing-pycon-india.md

```diff
@@ -5,6 +5,18 @@ Category: 2021
 Slug: pycon-india-announcement
 Summary: Announcing the return of PyCon India as an online conference for 2021.
 
+** *There have been some updates to the conference structure*
```

Comment:
May be mention here, Earlier was the following dates, Now it is changed to following. so people does not get confused.

Reply:
Wouldn't that just add more dates to the already a little confusing mix?

Reply:
Better to be clear about it. We want to be going for clarity.

Reply:
I agree on not adding too many dates. People will get confused. But maybe rephrasing to something like "The updated dates for the conference are...". Though I am no content expert, so will leave it on content experts to figure out the actual copy.

Design wise, I would say to put the dates in bold to make them more prominent.

Another thing to note, which is not related to this PR and is probably worth creating an issue for. Seems like the font-size for the list items is much smaller compared to rest of the text, which kind of gives the notion that the details are some secondary information (which I am guessing it is not). I don't expect a change for that here, but maybe a separate PR to update the css.